speechx/speechx/decoder/ctc_prefix_beam_search_decoder.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 Mobvoi Inc (Binbin Zhang)
+// Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// modified from
+// https://github.com/wenet-e2e/wenet/blob/main/runtime/core/decoder/ctc_prefix_beam_search.cc
+
+#pragma once
+
+#include "decoder/ctc_beam_search_opt.h"
+#include "decoder/ctc_prefix_beam_search_score.h"
+#include "decoder/decoder_itf.h"
+#include "fst/symbol-table.h"
+
+namespace ppspeech {
+class ContextGraph;
+class CTCPrefixBeamSearch : public DecoderBase {
+  public:
+    explicit CTCPrefixBeamSearch(const std::string vocab_path,
```

Comment:
```suggestion
   CTCPrefixBeamSearch(const std::string vocab_path,
```

构造函数有多个参数，不需要加 explicit.

Reply:
explicit 应该是禁止默认转换吧？多个参数也可以的吧？

Reply:
一般只用于 单个参数.

如果你要用于多个参数的话，为了代码风格统一起见，其他的 constructor, 是不是也需要加一个 explicit?